pages/sections.tsx

```diff
@@ -37,6 +39,78 @@ export const SectionsPage: NextPage = () => {
     "tenant"
   ]);
 
+  const idAndResidencyStartedValidator = (
+    values: [
+      StoreValue<
```

Comment:
I think this is an array of values, not a single value in an array. You express that as `Type[]` rather than `[Type]`.